opencv_apps/CMakeLists.txt

```diff
@@ -70,6 +70,10 @@ catkin_package()
 include_directories( ${catkin_INCLUDE_DIRS} ${OpenCV_INCLUDE_DIRS})
 
 ## Declare a cpp library
+if("${OpenCV_LIB_COMPONENTS}" MATCHES ";opencv_contrib;")
```

Comment:
on indigo/opencv2 environment, we do not have opencv_optflow nor TARGET openecv_optflow